config/set/dead-code.php

```diff
@@ -70,6 +71,7 @@
         RemoveAndTrueRector::class,
         RemoveConcatAutocastRector::class,
         SimplifyUselessVariableRector::class,
+        SimplifyUselessLastVariableAssignRector::class,
```

Comment:
Please revert this, I or @TomasVotruba can add it later and ensure that auto-commit Rectify (which can't runnable on fork) with added rule to the config set doesn't cause a regression on existing codebase.

Reply:
Ah ok, good to know!